### ISSUE TITLE
Strings with `\\` being altered to `\`

### ISSUE THREAD

**Issue by chasegiunta** [trivago/prettier-plugin-twig-melody#11]
```
{{ someMethod('foo\\bar\\baz') }}
```
will format to
```
{{ someMethod('foo\bar\\baz') }}
```

**Comment by twbartel** [comment]
On it...

**Comment by twbartel** [comment]
Hi @chasegiunta, I just published version 0.2.0 which should fix this issue. Can you have a look?

**Comment by chasegiunta** [comment]
Looks good, thank you!
